### ISSUE TITLE
After close with verticalGesture - remove the animation of hiding the browser

### ISSUE THREAD

**Issue by ilant** [thiagoperes/IDMPhotoBrowser#233]
After close the browser with verticalGesture - where is delay before the presenting view controller gets interactive (you can't tap or scroll it). Maybe need to set the animated param for "dismissPhotoBrowserAnimated" inside "doneButtonPressed()" function to "NO" when coming from "panGestureRecognized's .ended state" ?

**Comment by toby-pondr** [comment]
I agree. I'm not allowing the ability to close the gallery via swipe gesture until this is sped up. There is just that slight delay where the view controller presenting vc becomes interactive again
